app/src/androidTest/java/org/greenstand/android/TreeTracker/orgpicker/OrgPickerScreenKtTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.greenstand.android.TreeTracker.orgpicker
+
+import androidx.compose.ui.test.*
```

Comment:
When trying to run this test locally I couldn't get it working. Were you able to run this test?

Reply:
Yes